### PR TITLE
Add danish keymap

### PR DESCRIPTION
**Pull request by Nisker**
This adds a danish keymap, which is slightly different from Norwegian.
It also changes the shifted keys in the bottom to match danish shifted keys.
If you like i can make the other nordic layouts match their respective shifted keys aswell.

### CODE REVIEW

File: src/main/python/keymaps.py [hunk 1]
```diff
@@ -10,7 +10,8 @@
     ("Norwegian (QWERTY)", norwegian.keymap),
     ("Russian (ЙЦУКЕН)", russian.keymap),
     ("Spanish (QWERTY)", spanish.keymap),
-    ("Swedish (QWERTY)", swedish.keymap)
+    ("Swedish (QWERTY)", swedish.keymap),
+    ("Danish (QWERTY)", danish.keymap)
```
Comment (xyzz) [inline src/main/python/keymaps.py:14]: This should go before "French (AZERTY)", in alphabetical order

**Comment by xyzz** [comment]
>  If you like i can make the other nordic layouts match their respective shifted keys aswell.

Not sure what you mean, are the existing layouts wrong?

**Comment by Nisker** [comment]
> > If you like i can make the other nordic layouts match their respective shifted keys aswell.
> 
> Not sure what you mean, are the existing layouts wrong?

They are a little incomplete.
In the bottom of the basic tab you have the shifted keys where "(" is shift + 9 in US layout, but in most non-us layouts "(" it's shift + 8. 
So the shifted keys don't match.
It can be seen if you go to Norwegian layout and place a "(" the key will have the text "Lsft[ ) 9 ]"

**Comment by xyzz** [comment]
Ah yes, I wasn't sure yet what we should do about shifted keys. I think your suggestion is reasonable though.

**Comment by Nisker** [comment]
Then i will be doing the same thing for, the Swedish, Norwegian and maybe German.
I'll just open a new PR when they are done, if that's okay

**Comment by xyzz** [comment]
OK, sounds good.

**Pull request merged**